docs/source/tutorials/token_classification/entity_recognition_training.ipynb

```diff
@@ -0,0 +1,737 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "59db66ab",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture \n",
+    "!pip install transformers tqdm "
```

Comment:
sklearn and bert seem like two other dependencies

Reply:
I guess bert is just the script bert.py which you temporarily deleted making me confused, sorry

Reply:
yep, although as discussed I've recovered `bert.py`. `sklearn` added